Exercises/NoModules/Chapter 15/Soln15_02/Animals.cpp

```diff
@@ -0,0 +1,57 @@
+// Implementations of the Animal class and classes derived from Animal
+
+#include "Animals.h"
+
+// Constructor
+Animal::Animal(std::string_view name, unsigned weight) 
+  : m_name(name), m_weight(weight)
+{}
+
+// Return string describing the animal
+std::string Animal::who() const
+{
+  return "My name is " + m_name + ". My weight is " + std::to_string(m_weight) + " lbs.";
```

Comment:
format? would be nicer than to_string() and plusses. probably on next solutions as well if you change it